docs/spark-standalone.md

```diff
@@ -93,6 +93,10 @@ You can optionally configure the cluster further by setting environment variable
 <table class="table">
   <tr><th style="width:21%">Environment Variable</th><th>Meaning</th></tr>
   <tr>
+    <td><code>SPARK_MASTER</code></td>
```

Comment:
This property is not specifically about configuration files, and is deprecated. You would set `--master` now. That's why I say documenting it doesn't sound like the right thing, right?